flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -128,80 +123,71 @@
 
 	// ------------------------------------------------------------------------
 	//  MiniBatch Options
 	// ------------------------------------------------------------------------
 
-	public static final ConfigOption<Boolean> SQL_EXEC_MINIBATCH_ENABLED =
-			key("sql.exec.mini-batch.enabled")
-					.defaultValue(false)
-					.withDescription("Specifies whether to enable MiniBatch optimization. " +
-							"MiniBatch is an optimization to buffer input records to reduce state access. " +
-							"This is disabled by default. To enable this, users should set this config to true.");
-
-	public static final ConfigOption<String> SQL_EXEC_MINIBATCH_ALLOW_LATENCY =
-			key("sql.exec.mini-batch.allow-latency")
-					.defaultValue("-1 ms")
-					.withDescription("The maximum latency can be used for MiniBatch to buffer input records. " +
-							"MiniBatch is an optimization to buffer input records to reduce state access. " +
-							"MiniBatch is triggered with the allowed latency interval and when the maximum number of buffered records reached. " +
-							"NOTE: If " + SQL_EXEC_MINIBATCH_ENABLED.key() + " is set true, its value must be greater than zero.");
-
-	public static final ConfigOption<Long> SQL_EXEC_MINIBATCH_SIZE =
-			key("sql.exec.mini-batch.size")
-					.defaultValue(-1L)
-					.withDescription("The maximum number of input records can be buffered for MiniBatch. " +
-							"MiniBatch is an optimization to buffer input records to reduce state access. " +
-							"MiniBatch is triggered with the allowed latency interval and when the maximum number of buffered records reached. " +
-							"NOTE: MiniBatch only works for non-windowed aggregations currently. If " + SQL_EXEC_MINIBATCH_ENABLED.key() +
-							" is set true, its value must be positive.");
-
-	// ------------------------------------------------------------------------
-	//  State Options
-	// ------------------------------------------------------------------------
-	public static final ConfigOption<String> SQL_EXEC_STATE_TTL =
-			key("sql.exec.state.ttl")
-					.defaultValue("-1 ms")
-					.withDescription("Specifies a minimum time interval for how long idle state " +
-							"(i.e. state which was not updated), will be retained. State will never be " +
-							"cleared until it was idle for less than the minimum time, and will be cleared " +
-							"at some time after it was idle. Default is never clean-up the state.\n" +
-							"NOTE: Cleaning up state requires additional overhead for bookkeeping.");
+	public static final ConfigOption<Boolean> TABLE_EXEC_MINIBATCH_ENABLED =
+		key("table.exec.mini-batch.enabled")
+			.defaultValue(false)
+			.withDescription("Specifies whether to enable MiniBatch optimization. " +
+				"MiniBatch is an optimization to buffer input records to reduce state access. " +
+				"This is disabled by default. To enable this, users should set this config to true. " +
+				"NOTE: If mini-batch is enabled, 'table.exec.mini-batch.allow-latency' and " +
```

Comment:
use `TABLE_EXEC_MINIBATCH_ALLOW_LATENCY.key()` instead of `table.exec.mini-batch.allow-latency` ?  and `table.exec.mini-batch.size`

Reply:
We can't do that. Java will complain "Illegal forward reference" for that.